packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2591,9 +2610,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         excludeFromSemantics: true,
         axisDirection: _isMultiline ? AxisDirection.down : AxisDirection.right,
         controller: _scrollController,
-        physics: widget.scrollPhysics,
+        physics: widget.scrollPhysics ?? widget.scrollBehavior?.getScrollPhysics(context),
```

Comment:
Is this actually necessary? When `widget.scrollPhysics` is null, it should be the `Scrollable` that does this lookup?